src/call.c

```diff
@@ -2212,17 +2233,16 @@ int call_accept(struct call *call, struct sipsess_sock *sess_sock,
 		mem_deref(rid);
 	}
 
-	err = sipsess_accept(&call->sess, sess_sock, msg, 180, "Ringing",
+	err = sipsess_accept(&call->sess, sess_sock, msg, 180, "Trying",
 			     account_rel100_mode(call->acc),
 			     ua_cuser(call->ua), "application/sdp", NULL,
 			     auth_handler, call->acc, true,
 			     sipsess_offer_handler, sipsess_answer_handler,
 			     sipsess_estab_handler, sipsess_info_handler,
 			     call->acc->refer ? sipsess_refer_handler : NULL,
 			     sipsess_close_handler,
-			     call, "Allow: %H\r\n%H",
```

Comment:
What's the reasoning behind the removal of the Allow and Require header?

According to [RFC 3261 section 20](https://www.rfc-editor.org/rfc/rfc3261#section-20) they are optional in INVITE responses.